qa/tasks/cephfs/test_client_limits.py

```diff
@@ -43,6 +43,11 @@ def _test_client_pin(self, use_subdir):
         open_files = 200
 
         self.set_conf('mds', 'mds cache size', cache_size)
+        min_caps_per_client = CAP_RECALL_MIN
+	max_caps_per_client = int(cache_size * CAP_RECALL_RATIO)
+	if max_caps_per_client < min_caps_per_client
```

Comment:
Mixed tabs/spaces.

Reply:
@batrick updated it
I think it's the editor's question, thank you